### ISSUE TITLE
adding multiple bottle feedings results in thinking theyre overlapping

### ISSUE THREAD

**Issue by ocean-haiyang** [babybuddy/babybuddy#829]
If I add multiple bottling feedings with diff start times but add them in the same moment the program thinks theyre overlapping. I have to wait until a minute passes to add the next one.

**Comment by aryansanchiya** [comment]
Are you getting this "Error: Another entry intersects the specified time period. " ?

**Comment by ocean-haiyang** [comment]
> Are you getting this "Error: Another entry intersects the specified time period. " ?

Yes, even though it isn’t true. I think it’s based one when you enter it. Not the time period. The bottle entry feature is new so I think it’s caused there.

**Comment by cdubz** [comment]
@ocean-haiyang can you elaborate on what you mean by "add them in the same moment"?

In the demo instance I am able to add bottle entries in quick succession without the warning -- I don't have to wait a minute between entries. But I'm not sure I'm understanding your use case correctly.

https://github.com/babybuddy/babybuddy/assets/10456740/1c6b3995-498a-4741-851d-d9072f9d84ca

**Comment by Pino112** [comment]
> @ocean-haiyang can you elaborate on what you mean by "add them in the same moment"?
> 
> In the demo instance I am able to add bottle entries in quick succession without the warning -- I don't have to wait a minute between entries. But I'm not sure I'm understanding your use case correctly.
> 
>  Screen.Recording.2024-06-05.at.05.56.10.mov

The error occurs when you add a feeding entry that took place before one that you have already entered.

For example:
Add a feeding at 6pm, then add a feeding at 5pm. You will receive an error.

**Comment by KingPin** [comment]
same issue here, if you try to enter an entry from 7am, when you already have one from 9am, the 7am one will fail with "Error: Another entry intersects the specified time period."

**Comment by ocean-haiyang** [comment]
> @ocean-haiyang can you elaborate on what you mean by "add them in the same moment"?
> 
> In the demo instance I am able to add bottle entries in quick succession without the warning -- I don't have to wait a minute between entries. But I'm not sure I'm understanding your use case correctly.
> Screen.Recording.2024-06-05.at.05.56.10.mov

That was my same use case exactly. 

> > @ocean-haiyang can you elaborate on what you mean by "add them in the same moment"?
> > In the demo instance I am able to add bottle entries in quick succession without the warning -- I don't have to wait a minute between entries. But I'm not sure I'm understanding your use case correctly.
> > Screen.Recording.2024-06-05.at.05.56.10.mov
> 
> The error occurs when you add a feeding entry that took place before one that you have already entered.
> 
> For example: Add a feeding at 6pm, then add a feeding at 5pm. You will receive an error.

That is exactly the issue!

**Comment by cdubz** [comment]
Flagging #883 as closelt related (I think not quite duplicate). I'll try to looks at these bugs next time I'm able to spend some time on Baby Buddy. In the meantime PRs welcome as always 😁 

🐛 🐛 🐛
